CONTRIBUTING.md

```diff
@@ -2,6 +2,12 @@
 
 Thank you for your interest in contributing to Taichi! Please check out the [Contribution Guidelines](https://docs.taichi.graphics/lang/articles/contribution/contributor_guide) for how to make a contribution.
 
-## Developer Installation
+## Developer installation
 
 Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from source. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform.
+
+## Contribution opportunities
+
+Issues marked with ["welcome contribution"](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22welcome+contribution%22) are great places for starters. You can quickly get an idea of the entire workflow and how to join the community.
```

Comment:
good first issue or welcome contribution?  ;)

Reply:
"welcome contribution" is for developers who wish to dive deeper into Taichi; "good first issue" for developers new to Taichi.